docs/aliases/awful_client.lua

```diff
@@ -0,0 +1,6 @@
+---------------------------------------------------------------------------
+--- This module is deprecated, use `client`
+--  ===============================
+--
+-- @module awful.client
+---------------------------------------------------------------------------
```

Comment:
Not that it is important (feel free to ignore this comment), but this commit could also add that page to the ldoc config.